pkg/component/observability/monitoring/kubestatemetrics/resources.go

```diff
@@ -391,6 +391,8 @@ var gardenMetricAllowlist = []string{
 	"^kube_customresource_verticalpodautoscaler_spec_resourcepolicy_containerpolicies_maxallowed_cpu$",
 	"^kube_customresource_verticalpodautoscaler_spec_resourcepolicy_containerpolicies_maxallowed_memory$",
 	"^kube_customresource_verticalpodautoscaler_spec_updatepolicy_updatemode$",
+	"^garden_garden_condition$",
```

Comment:
I think the `garden_` prefix has to be replaced by `kube_customresource_` here (see VPA metrics above).

Reply:
I was wrong. The name prefix is part of the configuration. An incomplete cluster role was the reason for the metrics not being exported.

Reply:
You were faster to post this comment than my `make generate` command. I added a commit, the metrics are created and also scraped successfully by the `prometheus-garden` pods.